source/extensions/common/wasm/context.h

```diff
@@ -290,6 +293,15 @@ class Context : public Logger::Loggable<Logger::Id::wasm>,
   virtual WasmResult grpcSend(uint32_t token, absl::string_view message,
                               bool end_stream); // stream only
 
+  // CEL evaluation
+  virtual std::vector<const google::api::expr::runtime::CelFunction*>
+  FindFunctionOverloads(absl::string_view) const override {
```

Comment:
Functions start with a lower case letter.

Reply:
This is an override for a google-style declaration.